web_src/js/modules/init.ts

```diff
@@ -0,0 +1,27 @@
+export class InitPerformanceTracer {
+  results: {name: string, dur: number}[] = [];
+  recordCall(name: string, func: ()=>void) {
+    const start = performance.now();
+    func();
+    this.results.push({name, dur: performance.now() - start});
+  }
+  printResults() {
+    this.results = this.results.sort((a, b) => b.dur - a.dur);
+    for (let i = 0; i < 20 && i < this.results.length; i++) {
+      // eslint-disable-next-line no-console
+      console.log(`performance trace: ${this.results[i].name} ${this.results[i].dur.toFixed(3)}`);
+    }
```

Comment:
```suggestion
    for (const [index, {name, dur}] of this.results.entries()) {
      if (index >= 20) break;
      console.info(`performance trace: ${name} ${dur.toFixed(3)}`);
    }
```

Reply:
I do not think it is performant, by the `for ( let ...; i < len; ...)` loop, we do not need to create new "entries" object or extra `if` check